codechecker_common/review_status_handler.py

```diff
@@ -320,7 +320,7 @@ def get_review_status_from_source(
         # the source files are extracted as the original file path.
         # TODO: Should original_path be strip('/') at store?
         source_file_name = os.path.realpath(os.path.join(
-            self.__source_root, report.file.original_path))
+            self.__source_root, report.file.original_path.strip('/')))
```

Comment:
Question: Should this use `lstrip` instead of plain `strip`?

Reply:
Valid question, the goal is to make `report.file.original_path` not absolute. lstrip would only remove the forward slash from the front of the path string.